wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -62,6 +62,8 @@ public enum SemanticColors {
         static let textInactive = UIColor(light: Asset.Colors.gray60, dark: Asset.Colors.gray70)
         static let textParticipantDisconnected = UIColor(light: Asset.Colors.red300Light, dark: Asset.Colors.red300Dark)
 
+        // UserCell: e.g. "Paul Nagel (You)"
+        static let youSuffix = UIColor(light: Asset.Colors.gray70, dark: Asset.Colors.gray60)
```

Comment:
We have a format for semantic collars: '<usage>.<context/role>.<state?>'.
Documentation at the top of the class

Reply:
I'm not sure what the correct name of this property would be, I'll just prefix it with `text` like all the others.